libmscore/edit.cpp

```diff
@@ -446,7 +446,8 @@ bool Score::rewriteMeasures(Measure* fm, Measure* lm, const Fraction& ns, int st
 
             int tick1 = m1->tick();
             int tick2 = m2->endTick();
-            for (auto i : s->spannerMap().findContained(tick1, tick2))
+            auto spanners = s->spannerMap().findContained(tick1, tick2);
+            for (auto i : spanners)
                   undo(new RemoveElement(i.value));
```

Comment:
FWI, this was what I was concerned about when I thought maybe we might be removing from the list we were iterating on, depending on exactly how those loops are evaluated.  Probably there was never a problem because either findContained() is re-evaluated on each iteration, or the result of findContained() was stored in a temporary variable and removing from the score might be removed from the spanner map but the temporary variable wouldn't be affected.  Still it seemed awkward, so I just made this look like the others, first calling findContained() and *then* initiating the loop.

Same story with the call in joinMeasures() a little farther down.